packages/remix-dev/cli/commands.ts

```diff
@@ -56,7 +56,13 @@ export async function create({
   spinner.clear();
 }
 
-export async function init(projectDir: string) {
+type InitFlags = {
+  deleteScript?: boolean;
```

Comment:
just for clarity, should this be renamed to `deleteInitScript` ?

Reply:
We're calling the `init` command, so it's obvious that we're talking about the init script I think.